### PR TITLE
ZEPPELIN-657: Change the official location of Zeppelin in the documentation

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
Lately, an official location of Zeppelin was changed from **https://github.com/apache/incubator-zeppelin.git** to **http://git.apache.org/incubator-zeppelin.git**. So I changed the old location in several documentations to the latest one. 

### What type of PR is it?
Documentation

### Is there a relevant Jira issue?
[ZEPPELIN-657](https://issues.apache.org/jira/browse/ZEPPELIN-657)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: docs/development/howtocontribute.md [hunk 1]
```diff
@@ -5,52 +5,49 @@ description: "How to contribute"
...
-Oracle Java 7
-Apache Maven
+  
+  * Oracle Java 7
+  * Apache Maven
 
-### Getting the source code
-First of all, you need the Zeppelin source code. The official location for Zeppelin is [https://github.com/apache/incubator-zeppelin](https://github.com/apache/incubator-zeppelin)
+## Getting the source code
+First of all, you need Zeppelin source code. The official location of Zeppelin is [http://git.apache.org/incubator.git](http://git.apache.org/incubator-zeppelin.git).
```
Comment (felixcheung) [inline docs/development/howtocontribute.md]: `[http://git.apache.org/incubator.git](http://git.apache.org/incubator-zeppelin.git)`
I think this should be
`[http://git.apache.org/incubator-zeppelin.git](http://git.apache.org/incubator-zeppelin.git)`

**Comment by felixcheung** [comment]
shouldn't it be git: instead of http:?
git://git.apache.org/incubator-zeppelin.git

File: scripts/vagrant/zeppelin-dev/show-instructions.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ echo '# Post vagrant up instructions.'
 echo '# From your host machine,'
 echo '# git clone the incubator-zeppelin branch into this directory'
 echo
-echo 'git clone https://github.com/apache/incubator-zeppelin.git'
+echo 'git clone http://git.apache.org/incubator.git'
```
Comment (felixcheung) [inline scripts/vagrant/zeppelin-dev/show-instructions.sh]: same for this

File: scripts/vagrant/zeppelin-dev/README.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ curl -fsSL https://raw.githubusercontent.com/NFLabs/z-manager/master/zeppelin-in
 
 ### Building Zeppelin
 
-You can now `git clone https://github.com/apache/incubator-zeppelin.git` into a directory on your host machine, or directly in your virtual machine.
+You can now `git clone http://git.apache.org/incubator.git` into a directory on your host machine, or directly in your virtual machine.
```
Comment (felixcheung) [inline scripts/vagrant/zeppelin-dev/README.md]: same here: git.apache.org/incubator-zeppelin.git

File: docs/install/virtual_machine.md [hunk 4]
```diff
@@ -34,19 +34,22 @@ This script requires three applications, [Ansible](http://docs.ansible.com/ansib
...
-`sudo pip install ansible`  
-`ansible --version` should now report version 1.9.2 or higher
+2. Install Ansible:  [Ansible Python pip install](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip)
+    
+    ```
+    sudo easy_install pip
+    sudo pip install ansible
+    ansible --version 
+    ```
+    After then, please check whether it reports **version 1.9.2 or higher**.
```
Comment (felixcheung) [inline docs/install/virtual_machine.md]: I think we should say `ansible version 1.9.2...`

**Comment by AhyoungRyu** [comment]
@felixcheung yeah, you're right. I changed all of the locations and also applied it to `CONTRIBUTING.md`.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -144,13 +144,13 @@ First of all, you need the Zeppelin source code. The official location for Zeppe
 Get the source code on your development machine using git.
 
 ```
-git clone http://git.apache.org/incubator-zeppelin.git zeppelin
+git clone git://git.apache.org/incubator-zeppelin.git zeppelin
 ```
 
-You may also want to develop against a specific release. For example, for branch-0.1
+You may also want to develop against a specific release. For example, for branch-0.5.6
```
Comment (Leemoonsoo) [inline CONTRIBUTING.md]: Precisely, releases are available through tag in `v[X].[Y].[Z]` format. Let's change sentence from `... against a specific release. ...` to `... against a specific branch. ```

**Comment by AhyoungRyu** [comment]
@Leemoonsoo I changed them in two files and pushed again : )

**Comment by Leemoonsoo** [comment]
Thanks. LGTM

**Comment by felixcheung** [comment]
LGTM

**Comment by Leemoonsoo** [comment]
Thanks @AhyoungRyu I'm merging it.

**Pull request closed without merging**